src/test/kotlin/Actions/GetJsonResponseActionTest.kt

```diff
@@ -0,0 +1,26 @@
+package Actions
+
+import StructuredDataGenerators.JsonGenerator
+import httpstatus.HttpStatus
+import mocks.HTTPResponseBuilderMock
+import org.json.JSONObject
+import kotlin.test.*
+
+class GetJsonResponseActionTest {
+    @Test
+    fun `act method sets the correct properties on ResponseBuilder`() {
+        val jsonGenerator = JsonGenerator()
+        val action = GetJsonResponseAction(jsonGenerator)
+        val responseBuilder = HTTPResponseBuilderMock()
+
+        action.act(responseBuilder)
+
+        val jsonBody = JSONObject()
```

Comment:
nitpick: I like having "expected" (i.e., `expectedJsonBody`) in the variables to be extra clear.

Reply:
Better for readability, I agree!